src/wallet/coinselection.cpp

```diff
@@ -369,6 +356,11 @@ CAmount OutputGroup::GetSelectionAmount() const
     return m_subtract_fee_outputs ? m_value : effective_value;
 }
 
+CAmount SelectionResult::GetWaste(CAmount change_cost) const
+{
+    return GetSelectionWaste(m_selected_inputs, change_cost, m_target, m_use_effective);
+}
```

Comment:
Maybe move this under the global `::GetSelectionWaste`? Currently no problem but if you'd ever want to make it static.

Reply:
Done